<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>423</width>
    <height>432</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="connGroupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>421</width>
      <height>291</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic UI Semilight</family>
     </font>
    </property>
    <property name="title">
     <string>Connection settings</string>
    </property>
    <widget class="QLabel" name="textLabel5_2">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>201</y>
       <width>33</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Yu Gothic UI Semilight</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>P&amp;ort:</string>
     </property>
     <property name="buddy">
      <cstring>portSpinBox</cstring>
     </property>
    </widget>
    <widget class="QLineEdit" name="editPassword">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>120</y>
       <width>180</width>
       <height>30</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QSpinBox" name="portSpinBox">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>200</y>
       <width>180</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Yu Gothic UI Semilight</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="specialValueText">
      <string>Default</string>
     </property>
     <property name="minimum">
      <number>-1</number>
     </property>
     <property name="maximum">
      <number>65535</number>
     </property>
     <property name="value">
      <number>-1</number>
     </property>
    </widget>
    <widget class="QLineEdit" name="editHostname">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>160</y>
       <width>180</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="editUsername">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>80</y>
       <width>180</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="textLabel3">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>41</y>
       <width>109</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Yu Gothic UI Semilight</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Database Name:</string>
     </property>
     <property name="buddy">
      <cstring>editDatabase</cstring>
     </property>
    </widget>
    <widget class="QLabel" name="textLabel5">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>161</y>
       <width>76</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Yu Gothic UI Semilight</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&amp;Hostname:</string>
     </property>
     <property name="buddy">
      <cstring>editHostname</cstring>
     </property>
    </widget>
    <widget class="QLineEdit" name="editDatabase">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>40</y>
       <width>180</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="textLabel4">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>81</y>
       <width>75</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Yu Gothic UI Semilight</family>
       <pointsize>12</pointsize>
       <underline>false</underline>
      </font>
     </property>
     <property name="text">
      <string>&amp;Username:</string>
     </property>
     <property name="buddy">
      <cstring>editUsername</cstring>
     </property>
    </widget>
    <widget class="QLabel" name="textLabel4_2">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>121</y>
       <width>70</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Yu Gothic UI Semilight</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&amp;Password:</string>
     </property>
     <property name="buddy">
      <cstring>editPassword</cstring>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>250</y>
       <width>211</width>
       <height>18</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Yu Gothic UI Semilight</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>Connect to the default Database</string>
     </property>
    </widget>
   </widget>
   <widget class="QCommandLinkButton" name="commandLinkButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>330</y>
      <width>101</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic UI Semilight</family>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QCommandLinkButton{background-color: rgb(170, 255, 255);}

</string>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
    <property name="icon">
     <iconset resource="loginbtn.qrc">
      <normaloff>:/new/loginbtn/loginbtn.png</normaloff>
      <normalon>:/new/loginbtn/loginbtn.png</normalon>
      <activeoff>:/new/loginbtn/loginbtn.png</activeoff>
      <activeon>:/new/loginbtn/loginbtn.png</activeon>
      <selectedoff>:/new/loginbtn/loginbtn.png</selectedoff>
      <selectedon>:/new/loginbtn/loginbtn.png</selectedon>:/new/loginbtn/loginbtn.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>423</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="loginbtn.qrc"/>
 </resources>
 <connections/>
</ui>
